src/core/webview/ClineProvider.ts

```diff
@@ -145,6 +145,10 @@ export class ClineProvider
 	private pendingOperations: Map<string, PendingEditOperation> = new Map()
 	private static readonly PENDING_OPERATION_TIMEOUT_MS = 30000 // 30 seconds
 
+	private cloudOrganizationsCache: CloudOrganizationMembership[] | null = null
+	private cloudOrganizationsCacheTimestamp: number | null = null
+	private static readonly CLOUD_ORGANIZATIONS_CACHE_DURATION_MS = 5 * 60 * 1000 // 5 minutes
```

Comment:
Does this mean it takes 5 minutes after joining an organization for it to reflect in the extension?

Reply:
@roomote Only cache the organizations for 5 seconds. We're just trying to avoid spamming the call when we save settings and call the post function over and over again at the same time.